### PR TITLE
updated license section per legal advice

### PR DESCRIPTION
**Pull request by rgreenberg1**
will also need to update NM License

### CODE REVIEW

File: src/content/products/deepsparse/enterprise.mdx [hunk 1]
```diff
@@ -300,7 +300,7 @@ Use Case: A workload that might benefit from the elastic scheduler is one in whi
 
 ### License
 
-The Community Edition of the project's binary containing the DeepSparse Engine is licensed under the [Neural Magic Engine License.](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE-NEURALMAGIC)
-Example files and scripts included in this repository are licensed under the [Apache License Version 2.0](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE) as noted.
+The Community Edition of the project's binary containing DeepSparse is licensed under the [Neural Magic Engine License.](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE-NEURALMAGIC)
+Some source code, example files, and scripts included in the GitHub repository or directory are licensed under the [Apache License Version 2.0](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE) as noted.
```
Comment (jeanniefinks) [inline src/content/products/deepsparse/enterprise.mdx:304]:
```suggestion
Some source code, example files, and scripts included in the deepsparse GitHub repository or directory are licensed under the [Apache License Version 2.0](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE) as noted.
```

File: src/content/products/deepsparse/enterprise.mdx [hunk 2]
```diff
@@ -300,7 +300,7 @@ Use Case: A workload that might benefit from the elastic scheduler is one in whi
 
 ### License
 
-The Community Edition of the project's binary containing the DeepSparse Engine is licensed under the [Neural Magic Engine License.](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE-NEURALMAGIC)
-Example files and scripts included in this repository are licensed under the [Apache License Version 2.0](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE) as noted.
+The Community Edition of the project's binary containing DeepSparse is licensed under the [Neural Magic Engine License.](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE-NEURALMAGIC)
```
Comment (jeanniefinks) [inline src/content/products/deepsparse/enterprise.mdx:303]:
```suggestion
DeepSparse Community is licensed under the [Neural Magic DeepSparse License.](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE-NEURALMAGIC)
```

File: src/content/products/deepsparse/enterprise.mdx [hunk 3]
```diff
@@ -300,7 +300,7 @@ Use Case: A workload that might benefit from the elastic scheduler is one in whi
...
+Some source code, example files, and scripts included in the GitHub repository or directory are licensed under the [Apache License Version 2.0](https://github.com/neuralmagic/deepsparse/blob/main/LICENSE) as noted.
 
 [The Enterprise Edition](https://docs.neuralmagic.com/products/deepsparse-ent) requires a Trial License or [can be fully licensed](https://neuralmagic.com/legal/master-software-license-and-service-agreement/) for production, commercial applications.
```
Comment (jeanniefinks) [inline src/content/products/deepsparse/enterprise.mdx:306]:
```suggestion
[DeepSparse Enterprise](https://docs.neuralmagic.com/products/deepsparse-ent) requires a Trial License or [can be fully licensed](https://neuralmagic.com/legal/master-software-license-and-service-agreement/) for production, commercial applications.
```

**Review by jeanniefinks — APPROVED** [review]

File: src/content/products/deepsparse/enterprise.mdx (hunk 2 above)
Comment (rgreenberg1) [inline src/content/products/deepsparse/enterprise.mdx]: This needs to say Neural Magic DeepSparse Community License.

Comment (jeanniefinks) [reply src/content/products/deepsparse/enterprise.mdx]: We're just waiting on the second reviewer and this will be updated per the edits of this PR.

**Review by Beth-Kosis — APPROVED** [review]
These changes look fine.

**Pull request merged**